src/storage/file_manager.c

```diff
@@ -1443,10 +1443,11 @@ file_header_dump_descriptor (THREAD_ENTRY * thread_p, const FILE_HEADER * fhead,
 					&index_name, NULL) == NO_ERROR)
 	  {
 	    file_print_name_of_class (thread_p, fp, &fhead->descriptor.btree.class_oid);
-	    fprintf (fp, ", %s, ATTRID: %5d \n", index_name != NULL ? index_name : "*UNKNOWN-INDEX*",
+	    fprintf (fp, ", %s, ATTRID: %5d ", index_name != NULL ? index_name : "*UNKNOWN-INDEX*",
 		     fhead->descriptor.btree.attr_id);
 	  }
       }
+      fprintf (fp, "\n");
```

Comment:
```suggestion
	  }
      fprintf (fp, "\n");
      }
```

Reply:
I think placing line breaks outside of the scope { ... } would be more intuitive since they are present in every case. However, does it seem better to put them inside the scope?

Reply:
It's a trivial one. So, I'm not that against your choice. That's my "approval" means.

If you want to know my view, the way I see it, the curly-braces are for the whole case and the `break` also belongs to the case. So, the `break` should be in the braces as well.

It's as things in https://google.github.io/styleguide/cppguide.html#Switch_Statements, although it's not about our case for us. Just see the formats.